inference-engine/src/transformations/src/transformations/op_conversions/convert_broadcast3.cpp

```diff
@@ -11,50 +11,101 @@
 
 #include <ngraph/opsets/opset1.hpp>
 #include <ngraph/opsets/opset3.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
 
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertBroadcast3, "ConvertBroadcast3", 0);
 
-void ngraph::pass::ConvertBroadcast3::convert_broadcast3() {
-    auto broadcast = std::make_shared<pattern::op::Label>(element::f32, Shape {}, pattern::has_class<opset3::Broadcast>());
+bool make_compatible_shape(const ngraph::PartialShape & input_shape, std::vector<size_t> & target_shape) {
+    if (input_shape.rank().is_dynamic()) {
+        return false;
+    }
+    const int64_t & input_shape_rank = input_shape.rank().get_length();
```

Comment:
I don't see ay problem in `const &` for variables.
`const`  is needed because I don't expect to change this variable.
'&' is needed for cases if `get_length` method start producing `const &` values.
I'm not agree with this changes. If you want we can discuss it offline but it's not a blocker for merge.

Reply:
No it is a nitpick. But here you don't have any advantage of using reference for int, just you should write more code.